packages/common/src/TransactionUtils.js

```diff
@@ -13,6 +13,10 @@ const ynatm = require("@umaprotocol/ynatm");
  * @return Error and type of error (originating from `.call()` or `.send()`) or transaction receipt and return value.
  */
 const runTransaction = async ({ web3, transaction, transactionConfig, availableAccounts = 1 }) => {
+  // Add chainId in case RPC enforces transactions to be replay-protected, (i.e. enforced in geth v1.10,
```

Comment:
I raised this issue to the Infura forum [here](https://community.infura.io/t/polygon-rpc-only-replay-protected-eip-155-transactions-allowed-over-rpc/2997) but this is something we should be setting anyways according to [this](https://blog.ethereum.org/2021/03/03/geth-v1-10-0/)

Reply:
when would these chainIds not match? web3 is initialized using the providers how could they break?

Reply:
nvm. I see it's not about them mismatching but rather due to replay protection.

Reply:
Yeah its all about replay protection, and it looks like geth is trying to get more users to start specifying `chainId` going forward

Reply:
Interesting. I noticed this on our own node, and didn't know what to do about it. +1